docs/internet-identity-spec.adoc

```diff
@@ -520,6 +522,7 @@ This flow is the boring default
 3. User sees their management screen. In particular
 +
 - Their user number
+- A button to add additional devices
 - The list of their devices, with device aliases, and a button to remove
```

Comment:
```suggestion
- The list of their devices, with device aliases, a symbol marking recovery devices, and a button to remove
```

Reply:
In the longterm I'd like to show recovery devices in a different list/menu, but you're right for now this seems like the simplest thing.

Reply:
Yeah, or that. As long as they are visible and distinguishable.